docs/streaming-kinesis-integration.md

```diff
@@ -9,7 +9,7 @@ Here we explain how to configure Spark Streaming to receive data from Kinesis.
 
 #### Configuring Kinesis
 
-A Kinesis stream can be set up at one of the valid Kinesis endpoints with 1 or more shards per the following
+A Kinesis stream can be set up at one of the valid Kinesis endpoints with 1 or more shards following
```

Comment:
Yes, but that's only removing "following" and I don't think that's an improvement.